db/storage/admin-files.ts

```
@@ -32,19 +23,28 @@ const uploadAdminFile = async (
     throw new Error(`File must be less than ${sizeLimitMB}MB`);
   }
 
-  const filePath = `${payload.user_id}/${Buffer.from(payload.file_id).toString('base64')}`;
+  // Generate upload URL using Convex
+  const uploadUrl = await convex.mutation(api.fileStorage.generateUploadUrl, {
+    serviceKey: process.env.CONVEX_SERVICE_ROLE_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)